Sources/App/Commands/Ingest.swift

```diff
@@ -185,6 +186,21 @@ func updateRepository(on database: Database,
         throw AppError.genericError(repository.package.id,
                                     "repository metadata is nil for package \(repository.name ?? "unknown")")
     }
+    
+    var fork: Fork?
+    do {
+        if let parentUrl = metadata.repository?.normalizedParentUrl {
+            if let packageId = try await Package.query(on: database)
+                .filter(\.$url == parentUrl)
```

Comment:
We'll want to make sure to do a case-insensitive compare here:

```suggestion
                .filter(\.$url, .custom("ilike"), parentUrl)
```

Reply:
Good catch! Thanks.